src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,81 @@
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../utils/PersonLink';
+import classNames from 'classnames';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+
+  const findPersonByName = (name: string) => {
+    return people.find(person => person.name === name);
+  };
```

Comment:
Performance issue: The function `findPersonByName` is called for each person in the `people` array. This results in a time complexity of O(n^2), which can significantly slow down the application when dealing with large datasets. Instead, you could create a mapping from names to persons once and use it to find the mother and father for each person. This would reduce the time complexity to O(n).